ecosystem/typescript/sdk/src/transaction_builder/index.ts

```diff
@@ -87,3 +86,6 @@ export class TransactionBuilderMultiEd25519 extends TransactionBuilder<SigningFn
     return bcsToBytes(this.signInternal(rawTxn));
   }
 }
+
+export * as BCS from './bcs';
```

Comment:
could you move the code to its own file, and leave `index.ts` as just a list of exports?

Reply:
👍

Reply:
Updated